index.js

```diff
@@ -33,28 +32,32 @@ module.exports = postcss.plugin('postcss-resolution-independence',
 		} = {}) => {
 	const minScaleFactor = minSize / baseSize;
 
-	return (css) => {
-		css.walkDecls(decl => {
-			const nodes = parser(decl.value, {loose: true}).parse()
-			nodes.walkNumberNodes(node => {
-				const value = parseFloat(node.value)
-				// The standard unit to convert (if no unit, we assume the base unit)
-				if (node.unit === unit) {
-					const scaledValue = Math.abs(value * minScaleFactor);
-					if (scaledValue && scaledValue <= minUnitSize) {
-						if (Math.abs(value) >= minUnitSize) {
-							node.value = minUnitSize * (value < 0 ? -1 : 1);
+	return {
+		postcssPlugin: 'postcss-resolution-independence',
+		Once (css) {

```

Comment:
According to the migration documentation, we could use `Declaration()` instead of this. Is it right?

Reply:
A one TC fail occurred, I think `Declaration()` will be replaced later(not this PR).

Reply:
🆗  Let's make a ticket.